addons/website/views/snippets/s_alert.xml

```diff
@@ -11,20 +11,30 @@
 </template>
 
 <template id="s_alert_options" inherit_id="website.snippet_options">
-    <xpath expr="." position="inside">
-        <div data-selector=".s_alert">
-            <we-colorpicker string="Color"
-                data-select-style="true"
-                data-css-property="background-color"
-                data-color-prefix="alert-"/>
+    <xpath expr="//div[@id='so_width']" position="before">
+        <div data-selector=".s_alert" data-js="Alert">
+            <we-select string="Type" data-apply-to=".fa.s_alert_icon">
+                <we-button data-select-class="fa-user-circle" data-trigger="s_alert_colorpicker" data-trigger-value="primary">Primary</we-button>
```

Comment:
You can put the `data-trigger` on the `we-select` if I implemented this correctly. So that you won't have to repeat it for each button.

Reply:
Indeed :+1: well implemented :smile_cat: